dev/run-tests.py

```diff
@@ -253,9 +253,11 @@ def kill_zinc_on_port(zinc_port):
     """
     Kill the Zinc process running on the given port, if one exists.
     """
-    cmd = ("/usr/sbin/lsof -P |grep %s | grep LISTEN "
-           "| awk '{ print $2; }' | xargs kill") % zinc_port
-    subprocess.check_call(cmd, shell=True)
+    cmd = "%s -P |grep %s | grep LISTEN | awk '{ print $2; }' | xargs kill"
+    try:
+        subprocess.check_call(cmd % ("lsof", zinc_port), shell=True)
+    except:
+        subprocess.call(cmd % ("/usr/sbin/lsof", zinc_port), shell=True)
```

Comment:
Maybe, `subprocess.call` -> `subprocess.check_call`?

Reply:
I intentionally use `subprocess.call` to continue the execution even if `lsof` and `/usr/sbin/lsof` do not exist. This is because it is ok for other steps if we fail to kill `zinc`.

WDYT?

Reply:
Hm, but it changes what originally `kill_zinc_on_port` does though because now it is not guaranteed to kill it. I see the point but let's stick to the original behaviour.

Reply:
I see. Since this change is not strong preference, I will revert this change to keep the original behavior.